core/arch/arch.h

```diff
@@ -119,6 +119,9 @@ mixed_mode_enabled(void)
 #    define SCRATCH_REG4_OFFS XSI_OFFSET
 #    define SCRATCH_REG5_OFFS XDI_OFFSET
 #    define CALL_SCRATCH_REG DR_REG_R11
+#    define MC_IBL_REG xcx
+#    define MC_RETVAL_REG xax
```

Comment:
Wondering why we have two definitions for the same thing; can we have just one `RETVAL_REG`?

Reply:
I wanted to name it in a similar fashion to `SC_RETURN_REG`, hence the shortened struct name as a prefix to indicate which struct this should be used with.

Reply:
Okay, we can keep it as is.